views/metrilo-front-end-library.php

```diff
@@ -0,0 +1 @@
+<script type="text/javascript" src="<?php echo '//'. $this->tracking_library; ?>"></script>
```

Comment:
Where do you set `tracking_library`?
This will add our library as a blocking javascript. We don't want our clients to suffer performance penalties if our javascript library is not served. We have a specific asynchronous way of adding the library in the old Woo plugin.

Reply:
Tracking library is set in the constructor of the admin-settings class. Perhaps i should rename the class aswell.

Reply:
Missing answer on second part of the comment. Resolving this in favour of comment on the new version of the code.